src/app/components/header/header.component.html

```diff
@@ -32,10 +32,12 @@ <h1 class="title" routerLink="/home">Verseghy</h1>
         <a href="">Munkatársak</a>
         <a href="">Osztályok</a>
         <a href="">Dokumentumok</a>
+        <a href="https://klik035992001.e-kreta.hu/Adminisztracio/Login">E-Kréta</a>
       </div>
     </div>
   </div>
   <div class="left">
+    <a href="https://klik035992001.e-kreta.hu/Adminisztracio/Login" <img class="logo-image" src="assets/kreta.png"></a>
```

Comment:
First of all you can't nest html like this, so you can't do `<tag1 <tag2>>` in html.

If you want the image to be a clickable link, you should wrap the `img` tag around the `a` tag like this:
```
<a href="">
  <img src="">
</a>
```

Also, i think this is not the place for the KRETA button to be. I think it looks better before all the menu links like this (of course a bit smaller it was just a fast gimp):
![edit](https://user-images.githubusercontent.com/14909670/41740051-2ecb03ca-7597-11e8-899c-0979a6582472.png)

Secondly, the icons folder in the assets are reserved for those images so it's not a problem to place your kreta.png in the assets folder.

And there is a little problem with your image. It has a solid white background, so if we wanted to change the color of the header, we'd have to also change the image's background. So I think it's just better to use an image with transparent background like this one https://www.ekreta.hu/img/logo.png